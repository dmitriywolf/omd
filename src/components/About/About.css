.about-wrap {
  margin: 50px 0;
}

.about {
  display: flex;
  justify-content: space-between;
  align-items: flex-end;
  padding: 0 40px;
}

.about__img img{
  width: 300px;
  height: 300px;
  border-radius: 50%;
  overflow: hidden;
}

.about__content p {
  margin: 10px 0 0 0;
  font-size: 16px;
}

.about__content p span{
  font-weight: bold;
}
