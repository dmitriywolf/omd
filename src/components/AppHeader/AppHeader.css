.app__header {
  padding: 20px 0;
  background-color: rgba(4, 9, 3, 0.91);
}

.app__header-inner {
  display: flex;
  justify-content: space-between;
}
