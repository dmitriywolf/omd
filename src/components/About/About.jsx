import React from 'react';
import './About.css';
import img from './appImg.jpg';

const About = () => {

  return (
      <div className="about-wrap">
        <div className="container">
          <div className="about">

            <div className="about__content">
              <h3>Dmitriy Volchenko</h3>
              <p><a href="https://github.com/dmitriywolf">GitHub</a></p>
              <p><span>Age:</span> 25</p>
              <p><span>Country:</span> Ukraine</p>
              <p><span>Profession:</span> Mining Engineer</p>
              <p><span>Hard Skill:</span> HTML&CSS, JS, React</p>
            </div>

            <div className="about__img">
              <img src={img} alt="ВДС"/>
            </div>
          </div>
        </div>
      </div>
  )
};

export default About;
