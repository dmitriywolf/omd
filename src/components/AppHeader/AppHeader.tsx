import React from 'react';
import Menu from '../Menu';
import AppLogo from '../AppLogo';

import './AppHeader.css';

const AppHeader = () => {

  return (
    <header className="app__header">
      <div className="container app__header-inner">
        <AppLogo/>
        <Menu/>
      </div>
    </header>
  )
};

export default AppHeader;
