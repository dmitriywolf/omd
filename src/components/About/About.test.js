import React from 'react';
import About from './About';
import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";
import AppHeader from "../AppHeader";

describe("About component:", () => {
  it("render About component without crash", () => {
    shallow(<About/>);
  });

  it("should contain github link", () => {
    const wrap = shallow(<About/>);
    const link = <a href="https://github.com/dmitriywolf">GitHub</a>;
    expect(wrap.contains(link)).toEqual(true)
  });

  it("should render About component", () => {
    const component = shallow(<AppHeader/>);
    expect(toJson(component)).toMatchSnapshot();
  })
});


