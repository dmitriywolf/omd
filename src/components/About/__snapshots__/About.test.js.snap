// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component: should render About component 1`] = `
<header
  className="app__header"
>
  <div
    className="container app__header-inner"
  >
    <AppLogo />
    <Menu />
  </div>
</header>
`;
